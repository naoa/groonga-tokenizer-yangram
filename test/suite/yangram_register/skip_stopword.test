register tokenizers/yangram
yangram_register --skip_stopword 1
tokenizer_list
tokenize TokenYaBigramSkipStopword "今日は雨だ" NormalizerAuto
