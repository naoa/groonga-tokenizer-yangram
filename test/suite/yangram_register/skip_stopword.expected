register tokenizers/yangram
[[0,0.0,0.0],true]
yangram_register --skip_stopword 1
[[0,0.0,0.0],"TokenYaBigramSkipStopword"]
tokenizer_list
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "name": "TokenDelimit"
    },
    {
      "name": "TokenUnigram"
    },
    {
      "name": "TokenBigram"
    },
    {
      "name": "TokenTrigram"
    },
    {
      "name": "TokenBigramSplitSymbol"
    },
    {
      "name": "TokenBigramSplitSymbolAlpha"
    },
    {
      "name": "TokenBigramSplitSymbolAlphaDigit"
    },
    {
      "name": "TokenBigramIgnoreBlank"
    },
    {
      "name": "TokenBigramIgnoreBlankSplitSymbol"
    },
    {
      "name": "TokenBigramIgnoreBlankSplitSymbolAlpha"
    },
    {
      "name": "TokenBigramIgnoreBlankSplitSymbolAlphaDigit"
    },
    {
      "name": "TokenDelimitNull"
    },
    {
      "name": "TokenYaBigramSkipStopword"
    }
  ]
]
tokenize TokenYaBigramSkipStopword "今日は雨だ" NormalizerAuto
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "今日",
      "position": 0
    },
    {
      "value": "日は",
      "position": 1
    },
    {
      "value": "は雨",
      "position": 2
    },
    {
      "value": "雨だ",
      "position": 3
    },
    {
      "value": "だ",
      "position": 4
    }
  ]
]
